src/lib/db.ts

```diff
@@ -344,6 +367,51 @@ export class DB {
     });
   }
 
+  public createLearningLogWorkspace(title: string) {
+    const {user} = this.stores;
+
+    return new Promise<LearningLogWorkspaceModelType>((resolve, reject) => {
+      return this.createDocument("learningLog")
+        .then(({document, metadata}) => {
+          const learningLog: DBLearningLog = {
+            version: "1.0",
+            self: {
+              documentKey: document.self.documentKey,
+              uid: user.id,
+            },
+            title
+          };
+          return this.ref(this.getLearningLogPath(user)).push(learningLog).then(() => learningLog);
+        })
+        .then((learningLog) => {
+          return this.createWorkspaceFromLearningLog(learningLog);
+        })
+        .then(resolve)
+        .catch(reject);
+    });
+  }
+
+  public openLearningLogWorkspace(documentKey: string) {
```

Comment:
Seems like this function isn't called anywhere...are there future plans for it?

If it isn't necessary, is there a way to remove the equivalent `openSectionWorkspace` (which is called)?